docs/servo_power_monitor/src/graph.ts

```diff
@@ -1,26 +1,39 @@
 import Dygraph from 'dygraphs';
 import {Ui} from './ui';
+import {AnnotationText, PowerData} from './power_test_controller';
 
 export class Graph {
   private ui: Ui;
-  private g = new Dygraph('graph', [], {});
+  public g = new Dygraph('graph', [], {});
+  public annotations;
+  public annotationFlag = false;
+  private annotationText: AnnotationText = 'start';
   constructor(ui: Ui) {
     this.ui = ui;
+    this.annotations = this.g.annotations();
```

Comment:
Do we have to keep the previous annotations exists in the graph `this.g`? As the graph is created at the constructor, it should be empty, so this.annotations can just be an empty array (so we don't need this line.)